### PR TITLE
SW-5323 Updated types with new endpoints and updated module props

### PR DESCRIPTION
**Pull request by tommylau523**
This branch depends on https://github.com/terraware/terraware-server/pull/2336
Hoping to merge them together as well.

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#3106** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/3106?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/AcceleratorRouter/ParticipantProjects/Scoring/EditView.tsx [hunk 1]
```diff
@@ -83,7 +96,13 @@ const ScorecardEditView = () => {
   }, [updateStatus, goToScorecardView]);
 
   return (
-    <Page title={`Edit Scoring for project ${projectName}`} crumbs={crumbs} hierarchicalCrumbs={false}>
+    <PageWithModuleTimeline
+      title={`Edit Scoring for project ${project?.name ?? ''}`}
```
Comment (constanzauanini) [inline src/scenes/AcceleratorRouter/ParticipantProjects/Scoring/EditView.tsx:100]: We should localize this title (I know it's not form these changes)

**Comment by constanzauanini** [comment]
Lot of changes!
I'm not seeing the timeline in the terraware home page anymore. And then in the console side, in the project view only the phase is display but not the modules. (It used to display the modules there) Same for cohort and participant view.

**Comment by tommylau523** [comment]
> Lot of changes!
> I'm not seeing the timeline in the terraware home page anymore. And then in the console side, in the project view only the phase is display but not the modules. (It used to display the modules there) Same for cohort and participant view.
> 

This will depend on the BE changes going through first, since it is reacting to the API changes that are occuring.

File: src/scenes/AcceleratorRouter/ParticipantProjects/Scoring/EditView.tsx (hunk 1 above)
Comment (tommylau523) [reply src/scenes/AcceleratorRouter/ParticipantProjects/Scoring/EditView.tsx:100]: Looks like it is not actually used! Removing the title all together!

**Review by constanzauanini — APPROVED** [review]

**Comment by tommylau523** [comment]
### Merge activity

* **Aug 30, 3:31 PM EDT**: @tommylau523 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/3106).

**Pull request merged**